tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -796,7 +810,7 @@ export const renameResource = async (args: renameResourceArgs): Promise<void> =>
   }
 
   await page.locator(util.format(resourceNameSelector, resourceBase)).click({ button: 'right' })
-  await page.locator(util.format(filesAction, 'rename')).click()
+  await page.locator(util.format(filesBatchAction, 'rename')).click()
```

Comment:
I think this should be
```suggestion
  await page.locator(util.format(filesContextMenuAction, 'rename')).click()
```